wimprates/halo.py

```diff
@@ -79,41 +93,52 @@ def earth_velocity(t):
     phi = omega * (t - t1)
 
     # Mean orbital velocity of the Earth (Lewin & Smith appendix B)
-    v_orbit = 29.79 * nu.km / nu.s
+    v_orbit = _HALO_DEFAULTS['v_orbit'] * nu.km / nu.s
 
     v_earth_sun = v_orbit * (e_1 * np.cos(phi) + e_2 * np.sin(phi))
 
     # Velocity of Local Standard of Rest
-    v_lsr = np.array([0, 220, 0]) * nu.km/nu.s
+    v_lsr = np.array([0, v_0, 0])
     # Solar peculiar velocity
-    v_pec = np.array([11, 12, 7]) * nu.km/nu.s
+    v_pec = np.array(_HALO_DEFAULTS['v_pec']) * nu.km/nu.s
 
     return v_lsr + v_pec + v_earth_sun
 
 
 @export
-def v_earth(t=None):
+def v_earth(t=None, v_0=None, _n_days_average=365):
     """Return speed of earth relative to galactic rest frame
+    Velocity of earth/sun relative to gal. center (eccentric orbit, so not
+    equal to v_0).
+
+
     :param t: J2000 timestamp or None
+    :param v_0: Local standard of rest velocity
+    :param _n_days_average: if a v_0 is specified and t is not, average over
+    these many datapoints to get the year-averaged v_earth
     """
-    if t is None:
-        # Velocity of earth/sun relative to gal. center
-        # (eccentric orbit, so not equal to v_0)
+    # Float compares can be tricky, use np.isclose instead
+    if t is None and np.isclose(v_0, 220*nu.km/nu.s, atol=0, rtol=1e-8):
+        # Use old convention from https://arxiv.org/pdf/hep-ph/0504010.pdf
         return 232 * nu.km / nu.s
```

Comment:
```suggestion
```
I don't actually like this comparison as you might get different results while scanning v_0 and accidentally probing v_0 == 220 km/s. If you agree, we could just delete it and only rely on the t=59.37 day

Reply:
Good point, I agree. The loud warning should be enough to convince people to switch to an old wimprates if they want to use the old halo model without thinking/tinkering.

Reply:
great - I've removed this if statement